doc/getting-started.md

```diff
@@ -49,10 +45,10 @@ A very simple web form can be found under [upload](#/upload).
 
 #### Tag documentation
 
-After this you can tag a version, this can be useful when
+After uploading you can tag a specific version. This can be useful when
 the latest version should be available as `http://localhost:8000/docs/awesome-project/latest`
 
-If you want to tag the version `1.0.0` as `latest` for awesome-project.
+To tag the version `1.0.0` as `latest` for `awesome-project`:
 
 ```sh
 curl -X PUT http://localhost:8000/api/awesome-project/1.0.0/tags/latest
```

Comment:
Maybe we could add the docatl command as an alternative as well?
Same for the other docs as well.

Reply:
Fixed in #316